source/main/gameplay/SceneMouse.cpp

```diff
@@ -171,7 +171,7 @@ bool SceneMouse::mouseMoved(const OIS::MouseEvent& _arg)
 
             for (std::vector<hook_t>::iterator it = grab_truck->ar_hooks.begin(); it != grab_truck->ar_hooks.end(); it++)
             {
-                if (it->hk_hook_node->id == minnode)
+                if (it->hk_hook_node->pos == minnode)

```

Comment:
I think `pos` is correct here.

Reply:
Since `minnode` is an index, `pos` is definitely correct here.